engine/src/main/scala/cromwell/engine/workflow/lifecycle/MaterializeWorkflowDescriptorActor.scala

```diff
@@ -29,7 +29,9 @@ object MaterializeWorkflowDescriptorActor {
 
   val RuntimeBackendKey: String = "backend"
 
-  def props(serviceRegistryActor: ActorRef, workflowId: WorkflowId): Props = Props(new MaterializeWorkflowDescriptorActor(serviceRegistryActor, workflowId))
+  def cromwellBackends = CromwellBackends.instance.get
```

Comment:
In short, if this throws then `CromwellBackends.evaluateIfInitialised` would also have thrown. I'm relying here on standing on the shoulders of giants who would never let us get into such a sorry state... 🙏